error_reporting/google/cloud/error_reporting/util.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for Stackdriver Error Reporting."""
+
+from google.cloud.error_reporting.client import HTTPContext
+
+
+def build_flask_context(request):
```

Comment:
Only module besides init is client, was thinking client is core client code for all uses cases, this module has helpers for various random integrations like Flask. /shruggie though.